packages/console/src/containers/AppContent/components/Topbar/TenantSelector/components/TenantEnvTag/index.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+
+import DangerousRaw from '@/components/DangerousRaw';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  tag: string;
+  className?: string;
+};
+
+const tagTextMapping = (tag: string) => {
+  if (tag === 'development') {
+    return 'Dev';
+  }
+
+  if (tag === 'staging') {
+    return 'Staging';
+  }
+
+  if (tag === 'production') {
+    return 'Prod';
+  }
+
+  throw new Error(`Unsupported tag: ${tag}`);
```

Comment:
Would like to confirm whether we support anonymous tags or not. 
- if supported should not throw an error? Maybe show the tag name with a prefix
- if not, better define an enum to guard the value.

Reply:
Does not support anonymous tags. Temporarily use tag value directly. Will add a TenantTag to display value mapping once [PR #4004](https://github.com/logto-io/logto/pull/4004) is merged.